hobbify/habits/models/recurrent_instances.py

```diff
@@ -0,0 +1,33 @@
+""" Instance model """
+
+# Django
+from django.db import models
+
+# Models
+from hobbify.habits.models import Habit
+
+# Utils
+from hobbify.utils.models import HobbifyModel
+
+
+class RecurrentInstance(HobbifyModel):
+    """
+    An recurrent instance is an instance of a habit that
+    will follow the rrule, only past instances
+    will be stored in database.
+    """
```

Comment:
This is a weird style of docstring, but kind of looks consistent.

ditto: please automate code style verification

[Google Python Style Guide on class docstrings](https://google.github.io/styleguide/pyguide.html#384-classes)